### PR TITLE
feat(holodex): init

### PR DESCRIPTION
**Pull request by Guaxinim5573**
## 🎉 Theme for [Holodex](https://holodex.net) 🎉
[Holodex.net](https://holodex.net) is the website built by fans for fans to watch all their favorite virtual youtubers.

mocha

## 💬 Additional Comments 💬
Holodex's selected theme is ignored.

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/scripts/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - `catppuccin.user.css` - all the CSS for the userstyle, based on the
    template.
  - `assets/mocha.webp`, `assets/macchiato.webp`, `assets/frappe.webp` and
    `assets/latte.webp` - individual screenshots of the themed website, in all 4
    Catppuccin flavors.
  - `catwalk.webp` - image of all individual screenshots stitched together,
    generated via [Catwalk](https://github.com/catppuccin/toolbox/tree/main/catwalk#readme).

### CODE REVIEW

File: styles/holodex/catppuccin.user.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    div.v-slide-group__wrapper,
+    div.v-bottom-navigation,
+    div.v-select__slot,
+    div.v-input__slot,
+    div.v-overlay__scrim {
+      background: @base !important;
+    }
+    div.v-input {
+      padding: 0.2rem;
+    }
```
Comment (isabelroses) [inline styles/holodex/catppuccin.user.css:68]: Can you remove this line? Or does it fix some issue.

**Review by isabelroses — CHANGES_REQUESTED** [review]
First of all thanks for the PR, but there will need to be some changes.

- Text is not themed on most pages.
- It would be nice if streamers names where colored with the accent.
- We should also theme the red colors, in the image below.
image

**Comment by Guaxinim5573** [comment]
I've updated the theme with changes you requested.
image

Now Holodex's selected theme is ignored in favor of Catppuccin's accent color on Stylus.
<details><summary>Latte</summary></details>
<details><summary>Mocha</summary></details>

File: styles/holodex/catppuccin.user.css [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+    
+    & {
```
Comment (isabelroses) [inline styles/holodex/catppuccin.user.css:55]: we don't need to have the `&{}` here

**Review by isabelroses — CHANGES_REQUESTED** [review]
Just 1 last change and this PR should be good to go.

**Comment by isabelroses** [comment]
Thanks for the pr <3

**Pull request merged**